ShopperInsights/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -66,18 +69,34 @@ class ShopperInsightsClient @VisibleForTesting internal constructor(
             return
         }
 
-        paymentReadyAPI.processRequest(request)
+        //TODO: get correct merchant ID from SDK
+        val merchantId = "MXSJ4F5BADVNS"
+        val countryCode = "US"
+        val currencyCode = "USD"
+
+        val result = paymentReadyAPI.processRequest(
+            ShopperInsightsApiRequest(
+                request,
+                merchantId = merchantId,
+                currencyCode = currencyCode,
+                countryCode = countryCode
+            )
+        )
         // Hardcoded result
         callback.onResult(
             ShopperInsightsResult.Success(
                 ShopperInsightsInfo(
-                    isPayPalRecommended = false,
-                    isVenmoRecommended = false
+                    isPayPalRecommended = isPaymentRecommended(result.eligible_methods.paypal),
```

Comment:
please update the tests for this logic

Reply:
I will fix the unit tests at the end once we are all happy with the rest of the changes.

Reply:
fixed tests. I just made it one unit test for json to string conversion.